src/external/awslogs.cr

```diff
@@ -16,6 +16,10 @@ module Wsman
         Dir.mkdir_p(@config.awslogs_config_path)
         File.join(@config.awslogs_config_path, "wsman-#{site_name}.conf")
       end
+
+      def cleanup_site(site_name)
```

Comment:
Since all this function does is removes the site configuration, please follow the naming convention of other methods an rename it to something like delete_site_config.